src/roles/common/tasks/fix-image-paths.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Get unzipped files dir
+  set_fact:
+    nuage_unzipped_files_dir: "{{ common.nuage_unzipped_files_dir }}"
+
+- name: Replace non-relative path
+  set_fact:
+    nuage_unzipped_files_dir: "{{ nuage_unzipped_files_dir | regex_replace('.+?(?=metroae_data)', '/') }}"
+  when: common.nuage_unzipped_files_dir | regex_search("/metroae_data/")
+
+- name: Replace relative path
+  set_fact: 
+    nuage_unzipped_files_dir: "{{ '/metroae_data/' + nuage_unzipped_files_dir }}"
+  when: common.nuage_unzipped_files_dir | regex_search("^[^/]")
+

```

Comment:
If inside the container, let's also check if the path contains "/metroae_data/" and if not tell the user that for the container this directory needs to be specified under the /metroae_data/ installation mount directory from container installation.

Reply:
just to be sure, the path in this case is the path given by the common.nuage_unzipped_files_dir variable, not the path given by the {{ role_path }}?

Reply:
check nuage_unzipped_files_dir to make sure it starts with /metroae_data/.  Check if role_path starts with "/source/nuage-metroae" to know if this is running in a container.